src/TodoList.jsx

```diff
@@ -1,13 +1,24 @@
+import TodoListItem from "./TodoListItem";
+
 function TodoList() {
   const todos = [
-    { id: 1, title: "learn react in 12 weeks" },
-    { id: 2, title: "learn node in 13 weeks after react" },
-    { id: 3, title: "learn typescript whiles learning react" },
+    { id: 1, title: "learn react in 12 weeks", startDate: "11-03-25" },
+    {
+      id: 2,
+      title: "learn node in 13 weeks after react",
+      startDate: "02-06-25",
+    },
+    {
+      id: 3,
+      title: "learn typescript whiles learning react",
+      startDate: "18-03-25",
+    },
   ];
+
   return (
     <ul>
       {todos.map((todo) => (
-        <li key={todo.id}>{todo.title}</li>
+        <TodoListItem todo={todo} key={todo.id} startDate={todo.startDate} />
```

Comment:
Conventionally you want to have the `key` prop before all other props that are passed down since you cannot directly access it. This way it's not confused as a prop that is passed down to a child component.